lib/lotus/helpers/asset_tag_helpers.rb

```diff
@@ -0,0 +1,26 @@
+require 'lotus/helpers'
+
+module Lotus
+  module Helpers
+    # The helper methods used for using assets.
+    #
+    # @since 0.6.0
+    # @api public
+    module AssetTagHelpers
+      include Lotus::Helpers::HtmlHelper
+
+      def image(source, options = {})
+        raise ArgumentError.new("source argument can't be nil") if source.nil?
```

Comment:
@Nerian Because we're delegating source generation to `asset_path`, we can remove this guard from here.